tools/copy_engine_content.py

```diff
@@ -22,7 +22,10 @@
         os.path.realpath(os.path.join(args.unreal_engine_dir, "Samples", "StarterContent", "Content", "StarterContent")): \
             os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "StarterContent")),
         os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TemplateResources", "High", "Characters", "Content", "Mannequins")): \
-            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "Characters", "Mannequins"))}
+            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "Characters", "Mannequins")),
```

Comment:
Great. Do we need to add anything in here for the vehicle we're using in the car example?

Reply:
Our convention is to put the trailing `}` on the same line as the last entry in the dict.

```
    content_dirs = {
        os.path.realpath(os.path.join(args.unreal_engine_dir, "Samples", "StarterContent", "Content", "StarterContent")): \
            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "StarterContent")),
        os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TemplateResources", "High", "Characters", "Content", "Mannequins")): \
            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "Characters", "Mannequins")),
        os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TP_ThirdPersonBP", "Content", "ThirdPerson")): \
            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "ThirdPerson")),
        os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TemplateResources", "Standard", "Vehicles", "Content")): \
            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "Vehicles", )),
        os.path.realpath(os.path.join(args.unreal_engine_dir, "Templates", "TP_VehicleAdvBP", "Content", "VehicleTemplate")): \
            os.path.realpath(os.path.join(args.unreal_project_dir, "Content", "VehicleTemplate"))}
```